love/src/components/WeatherStation/WeatherStation.jsx

```diff
@@ -222,19 +237,50 @@ export default class WeatherStation extends Component {
           </div>
         </div>
 
+        {this.props.controls && (
+          <div className={styles.doubleSection}>
+            <div className={styles.sectionTitle}>Timeseries Controls</div>
+            <div className={styles.timeSeriesControls}>
+              <TimeSeriesControls
+                  setTimeWindow={(timeWindow) => this.setState({timeWindow})}
+                  timeWindow={this.state.timeWindow}
+                  setLiveMode={(isLive) => this.setState({isLive})}
+                  isLive={this.state.isLive}
+                  setHistoricalData={(historicalData) => this.setState({historicalData})}
+              />
+            </div>
+          </div>
+          )
+        }
+
+        {this.props.controls && (
+          <div className={styles.doubleSectionSimple}>
+            {this.state.isLive && <span>Showin last <strong>{this.state.timeWindow != 1 ? 
+              this.state.timeWindow + " minutes" : 
+              "minute"}</strong></span>}
+            {!this.state.isLive && <span>Showin from <strong>{this.state.historicalData?.[0]?.format(DATE_TIME_FORMAT)}</strong> to <strong>{this.state.historicalData?.[1]?.format(DATE_TIME_FORMAT)}</strong></span>}
```

Comment:
We have to use the time/day format as YYYY/MM/DD, and HH:MM:SS according to https://jira.lsstcorp.org/browse/LVV-14387. Also, there's a missing 'g' in 'Showing'. I suggest using 'Displaying data from' instead